src/util/bip341.rs

```diff
@@ -467,6 +459,14 @@ mod tests {
         );
     }
 
+    #[test]
+    fn sizeof() {
+        assert_eq!(size_of::<Option<CommonCache>>(), 97);
+        assert_eq!(size_of::<Option<TaprootCache>>(), 65);
+        assert_eq!(size_of::<&Transaction>(), 8);
```

Comment:
According to Andrew Poelstra, this is not the same across all machines. It is the same size as `usize`.

Reply:
Andrew is totally right!

I used the test for some reasoning on stack space usage, I think I can remove it.
done in a37de1a